packages/devtools_app/lib/src/flutter/notifications.dart

```diff
@@ -128,14 +130,21 @@ class NotificationsState extends State<_NotificationsProvider>
   Widget _buildOverlay(BuildContext context) {
     return Align(
       alignment: Alignment.bottomRight,
-      child: SizedBox(
-        width: _notificationWidth,
-        child: SingleChildScrollView(
-          reverse: true,
-          child: Column(
-            verticalDirection: VerticalDirection.down,
-            mainAxisSize: MainAxisSize.min,
-            children: _notifications,
+      child: Padding(
+        // Clear the status line.
```

Comment:
this comment is a bit misleading. This sounds like we are clearing the information from the status line. Perhaps replace with "Show notifications above the status line." or similar

Reply:
will re-word